delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -54,6 +54,7 @@ node {
 	def bitbucket_repo_name
 	def tracking_id
 	def config
```

Comment:
Do we need both config and configObj?

Reply:
Looking at the rest of your code, looks like configObj is your temp object for holding data from service module. So I would do 2 things: move the scope of this variable closer there. Also change the name to serviceObj

Reply:
In cooperated review comments